tests/classes/potentialSet.cpp

```diff
@@ -63,4 +64,154 @@ TEST(PotentialSetTest, ComplexAddition)
     EXPECT_EQ(2.0, pots.potentialMap()["A-D"].potential.value(0));
 }
 
+TEST(PotentialSetTest, Averaging)
+{
+    GenericList moduleData;
+    auto originalPotentialsObject =
+        moduleData.realiseIf<PotentialSet>("PotentialSet", "module", GenericItem::InRestartFileFlag);
+
+    std::string filename_{"test_restart.txt"};
+    // Open the file
+    LineParser parser;
+    if (!parser.openOutput(filename_))
+    {
+        parser.closeFiles();
+    }
```

Comment:
If we can't open the output file we can't do the test, so this should ASSERT:
```suggestion
    ASSERT_TRUE(parser.openOutput(filename_));
```